src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java

```diff
@@ -190,6 +199,31 @@
  *         the very same way as it is done by HTTPS. If "false" the host name is not verified at all. Per default the
  *         hostname is verified (true).
  *     </li>
+ *
+ *     <li>
+ *         <b>ch.hevs.cloudio.endpoint.token</b><br>
+ *
+ *         This property contains a single-use authentication token for the endpoint. This token is used to retrieve
+ *         certificates using the cloud.iO Rest API.
+ *
+ *     </li>
+ *     <li>
+ *         <b>ch.hevs.cloudio.endpoint.tokenPropertiesPath</b><br>
+ *
+ *         This property is used to specify a path where the retrieved certificates and .properties file from token
+ *         authentication will be stored. <br>
+ *
+ *         This property is optional and taken into account only if token authentication is used.
+ *         Its the default is <b>/etc/cloud.io/</b>.
+ *     </li>
+ *     <li>
+ *         <b>ch.hevs.cloudio.endpoint.apiUrl</b><br>
+ *
+ *          Main URL of the cloud.iO Rest API. This URL is used for the certificate retrieval using authentication
```

Comment:
Main URL of the cloud.iO Rest API. This URL is used for the certificate and settings retrieval using one-time provisioning with using a provisioning token.<br>